### ISSUE TITLE
Tab for indenting doesn't work inside links

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#755]
Example (`|` indicates cursor location):

```
- [Example |text](http://example.com/)
```

Pressing `Tab` moves the cursor after the bracket while it should indent the bullet point. Might be a regression in 4.0.0-rc8.

**Comment by astoilkov** [comment]
This is an intended behavior. When inside of an inline Markdown element Tab goes out of it. This works in bold, italic, links, etc. Tab also has a special meaning in tables.

**Comment by borekb** [comment]
Ah I see. Is there an easy to increase / decrease indent then? I Always used Tab and Shift+Tab, maybe I was always lucky not to be inside any inline element.

**Comment by astoilkov** [comment]
Yes. We have the shortcut available in other editors `Ctrl+]` and `Ctrl+[`. However, it is possible that you don't have the shortcut because sometimes when we add new shortcuts we don't introduce them to older users.

**Comment by borekb** [comment]
Oh that's why! I tried Ctrl+[ as in VSCode but it didn't do anything for me.

Funnily enough, just yesterday I was thinking about submitting a feature request for having `Preference.default.md` and `Keyboard Shortcuts.default.md` or similar in the config folder so that I could do a diff easily at any time. Those files would be just informative and Caret would not use them in runtime; they would be safe to overwrite with every release.
